recipes/atomicrex/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "atomicrex" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/{{ name }}/{{ name }}/-/archive/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f90d7a46987a363c2d14d693e7ac1d0e966b8a0a852aafe3254b30f006ea6555
+  patches:
+    - disabletests.patch
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - cmake 
+    - make
+    - python
```

Comment:
`python` should be in `host`

Reply:
The issue is they use the location of `cmake` to find the python executable. I know it is a bit strange and I am in contact with the developers to fix it, but for now I thought that adding python to the `build` section could solve this issue: https://gitlab.com/atomicrex/atomicrex/blob/master/cmake/Testing.cmake#L43

Reply:
I think you are misreading that line. CMAKE_BINARY_DIR is not where cmake is. It is the build directory and `${CMAKE_BINARY_DIR}/python` isn't for the python executable, it is the path to the python files which are in this directory called python